### PR TITLE
get_operation_id

### PR DESCRIPTION
**Pull request by Eitu33**

### CODE REVIEW

File: massa-execution-worker/src/interface_impl.rs [hunk 1]
```diff
@@ -842,6 +842,15 @@ impl Interface for InterfaceImpl {
         Ok(())
     }
 
+    // Returns the operation id that originated the current execution if there is one
+    fn get_origin_operation_id(&self) -> Result<Option<String>> {
+        let operation_id = context_guard!(self)
+            .origin_operation_id
+            .map(|op_id| op_id.to_string());
+        tracing::warn!("origin_operation_id: {:?}", operation_id);
```
Comment (damip) [inline massa-execution-worker/src/interface_impl.rs:850]: we should reduce the log level (eg. debug)

**Review by damip — APPROVED** [review]
LGTM

**Review by damip — APPROVED** [review]

**Comment by Eitu33** [comment]
bors merge

**Comment by AurelienFT** [comment]
Merge the latest version of main and it should work

**Comment by Eitu33** [comment]
bors merge

**Pull request merged**